<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="videoFrame">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>461</width>
      <height>521</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="videoLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>461</width>
       <height>521</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnKisiEkle">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Kİşi ekle</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnKisiSil">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>70</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>kişi sil</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOpenDir">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>161</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>open directory</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblPath">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>271</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>path:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnKaydet">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>480</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>kaydet</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listwidget">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>210</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::MultiSelection</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
